### DISCUSSION TITLE
Please tell me your favorite ice cream!!!

### DISCUSSION THREAD

**Discussion by Sachi-Ha** [NAIST-SE/AttractingContributors#854] [2023 Attraction Contributions]
I always eat same ice cream.
so I got tired of the taste.
Please tell me your favorite ice cream!!! 
↓↓
https://github.com/Sachi-Ha/AttractingContributors.git

**Comment by oto10** [comment]
I added my favorite ice cream to your list.
Could you please add your favorite music to my README.md?
https://github.com/oto10/AttractingContributors-your-favorite-music
Thank you in advance.

**Comment by zihang-wang-naist** [comment]
I added my favorite ice cream to your list.
Could you please add your favourite place to my README.md?
https://github.com/zihang-wang-naist/AttractingContribu-Travel
Thank you

**Comment by WEI44ZHEYUAN** [comment]
I added
Please help me
https://github.com/WEI44ZHEYUAN/AttractingContributors_Songs
